src/server.ts

```diff
@@ -38,8 +38,10 @@ export class HomebridgePluginUiServer {
     }
 
     process.addListener('message', (request) => {
```

Comment:
It looks like the type for `request` switched from `any` to `unknown`, which makes it more restrictive on how you can use it without manually casting. If you want to simply get back to the old behavior, you can specify the type as `any` here. That should clear both of the ts errors in this function.

```suggestion
    process.addListener('message', (request: any) => {
```